tripal_chado/src/Plugin/ChadoBuddy/ChadoCvtermBuddy.php

```diff
@@ -34,26 +111,73 @@ class ChadoCvtermBuddy extends ChadoBuddyPluginBase {
    *   If the select values return multiple records, then we return an array
    *     of ChadoBuddyRecords describing the results.
    *   If there are no results then we return FALSE and if an error is
-   *     encountered then an exception will be thrown.
+   *     encountered then a ChadoBuddyException will be thrown.
```

Comment:
Note: there is actually a drupal standard for documenting exceptions thrown :-) https://www.drupal.org/docs/develop/standards/php/api-documentation-and-comment-standards#throws To use it you would leave this as is but then add the `@throws` tag as well to make it easier to see as a developer.

Reply:
Adding @throws tags. There is also a @todo tag option, using that for the cvterm url comment!